src/dft/src/architect/ScanArchitectHeuristic.cpp

```diff
@@ -38,15 +38,84 @@
 #include "ClockDomain.hh"
 #include "ScanArchitect.hh"
 
+#include <boost/geometry.hpp>
+#include <boost/geometry/geometries/register/point.hpp>
+#include <boost/geometry/index/rtree.hpp>
+
+namespace bg = boost::geometry;
+namespace bgi = boost::geometry::index;
+
 namespace dft {
 
 ScanArchitectHeuristic::ScanArchitectHeuristic(
     const ScanArchitectConfig& config,
-    std::unique_ptr<ScanCellsBucket> scan_cells_bucket)
-    : ScanArchitect(config, std::move(scan_cells_bucket))
+    std::unique_ptr<ScanCellsBucket> scan_cells_bucket, utl::Logger* logger)
+    : ScanArchitect(config, std::move(scan_cells_bucket)), logger_(logger)
 {
 }
 
+void ScanArchitectHeuristic::order_reduce_wirelength(std::vector<std::unique_ptr<ScanCell>>& cells)
```

Comment:
method should follow Google style order_reduce_wirelength -> OrderReduceWirelength.

Reply:
To represent the functionality better I would suggest OptimizeScanWirelength, but that's more of a nit.